index.html

```diff
@@ -17,8 +19,9 @@ <h1>Welcome to the official page for the FCC Redmond, WA Chapter!</h1>
             <button>Let's Go!</button>
         </section>
         <section id="member-section">
-                <input type="text" id="skill-input" placeholder="enter one skill">
-                <input type="text" id="lName-input" placeholder="enter a last name">
+            <button id="add-member">Add</button>
```

Comment:
Is this Add button for adding new members? I think we should add new members in a different page so it's not as open to the public. The index page will just be for searching and filtering our member info.

Reply:
We can move it. Just wanted to test our CRUD operations, so added this code. Feel free to discard.